src/tests/ftest/soak/soak.yaml

```diff
@@ -189,8 +193,8 @@ ior_stress:
         - '512K'
         - '64k'
     daos_oclass:
-          - "SX"
-        # - "RP_2GX"
+        # - "SX"
```

Comment:
Is this intentional?  If so why not test both SX and RP_2GX?

Reply:
replication is not supported in 0.9 so I have them commented out for now.  Soak is mostly manual and you will tweek it as needed.